src/Tribe/Main.php

```diff
@@ -337,6 +337,9 @@ public function after_html_data_wrapper( $html ) {
 		 * Add filters and actions
 		 */
 		protected function addHooks() {
```

Comment:
Agreed, we could change it, but we might do some changes around the _"camelCase"_ on the next release, because of "breaking" changes. On this case it shouldn't break because of `private` but...